core_test.go

```diff
@@ -181,6 +181,54 @@ func TestMaxlagRetryFail(t *testing.T) {
 	}
 }
 
+func TestMultipartOffForSmallParameters(t *testing.T) {
+	smolH := strings.Repeat("h", 7500)
```

Comment:
`smolH`? How about `smallPayload` (and `bigPayload` for the variable in the other test, line 209).